### ISSUE TITLE
Train on RCTW dataset

### ISSUE THREAD

**Issue by Jacqueline720** [ZJULearning/pixel_link#84]
Has anybody trained pixel_link on the RCTW-17 from scratch? When I trained this dataset on pixel_link,the model can not converge:when I adjust learning rate as 1e-2,the loss goes to nan,while the loss goes to 0 when learning rate = 1e-3.
Hope anybody have any idea about this problem~

**Comment by HoracceFeng** [comment]
Hi @Jacqueline720, have you solved this problem?  In the source code, the image will be cropped to 512 while the images in RCTW dataset are dramatically larger, like 2000-3000 per edge. I have done some modification to make the whole image resize and put into training. But then I get the 'nan loss' once lr=1e-3. 

Hi author @dengdan, would you give me some suggestions?
Thx

**Comment by HoracceFeng** [comment]
Problem Solved. Many Questions are actually because of the TFrecords converter and mismatch of tf.summary.image(bounding_box). Will do a pipeline code for text segmentation later.

**Comment by yhpwsid** [comment]
> Hi @Jacqueline720, have you solved this problem? In the source code, the image will be cropped to 512 while the images in RCTW dataset are dramatically larger, like 2000-3000 per edge. I have done some modification to make the whole image resize and put into training. But then I get the 'nan loss' once lr=1e-3.
> 
> Hi author @dengdan, would you give me some suggestions?
> Thx
does the loss have anything to do with tf.summary.image? how did you solve the loss problem? thanks

**Comment by xiiiiiiii** [comment]
> Has anybody trained pixel_link on the RCTW-17 from scratch? When I trained this dataset on pixel_link,the model can not converge:when I adjust learning rate as 1e-2,the loss goes to nan,while the loss goes to 0 when learning rate = 1e-3.
> Hope anybody have any idea about this problem~

你好，想问一下你在RCTW-17数据集上准确率召回率是多少，表现如何
